TODO.md

```diff
@@ -68,3 +68,5 @@
   every read/write range command.
 
 * Gossip should be changed to send only protobufs.
+* persist Gossip and the kv.leaderCache to make sure that restarting
+  a node is harmless.
```

Comment:
I didn't realize each pair of nodes exchanged the complete gossip map. Seems like we may want some sort of rsync-like delta scheme for this so we don't have to use so much bandwidth even when a node is brand new (or its stored state is too old).

In bigtable there was no separate leader election so a scan of the metadata table was enough to find the location of each range. I think we should do something similar. I know there is no place to read leader info right now but I think it would be a good idea to write it into the meta keyspace (on a best-effort basis after the actual leader election has completed).